src/components/CampsiteListItem.js

```diff
@@ -3,102 +3,115 @@ import React from 'react';
 import {View, Text, StyleSheet, ScrollView, ImageBackground} from 'react-native';
 import {Card, ListItem, Button, Tile, List, Image, Badge, Icon} from 'react-native-elements';
 // 3rd party libraries - additional
+
 // styles and language
-import { badgeGreen, limeGreenTitle, RoyalBlueHighlight, blueGreenNav} from '../styles/index';
-import { campsite } from '../locale.en'
+import {featureIconDetails, facilityIconDetails} from '../constants';
+import {badgeGreen, limeGreenTitle, RoyalBlueHighlight, blueGreenNav} from '../styles/index';
+import {campsite} from '../locale.en'
 import _ from 'lodash';
 
 
-const CampsiteListItem = ({site}) => {
-    const{textStyle, IconContainer, subtitleView,siteAvatarStyle, siteAvatarContainerStyle, titleView, nearestTownStyle, badgeWrapperStyle, badgeContainerStyle,  accessibilityStyle} = styles
-    const {campsite_form:{accessibility_options}} = campsite;
-    const {id, title, description, nearestTown, accessibility, siteImageData} = site;
-return(
-    <ListItem
-            avatar={siteImageData ? {uri:`data:image/png;base64,${siteImageData}`} : require('../../assets/starTent.jpg')}
+const renderIcons = ({features, facilities}) => {
+    const featureIcons = _.map(features, feature =>{
+        return(
+            <Icon
+                key={feature}
+                name={featureIconDetails[feature].name}
+                type={featureIconDetails[feature].type}
+            />
+        )
+    });
+
+    const facilityIcons = _.map(facilities, facility =>{
+       return(
+         <Icon
+             key={facility}
+             name={facilityIconDetails[facility].name}
+             type={facilityIconDetails[facility].type}
+         />
+       );
+    });
+
+    return _.concat(featureIcons, facilityIcons);
```

Comment:
Dynamic icon rendering using the `featureIconDetails` and `facilityIconDetails` constants that are now in `constants.js`